app/containers/markdown/MessageBody/Heading.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Text } from 'react-native';
+import PropTypes from 'prop-types';
+import { themes } from '../../../constants/colors';
+import styles from '../styles';
+import { useTheme } from '../../../theme';
+
+const Heading = ({ value, level }) => {
+	const { theme } = useTheme();
+	const textStyle = styles[`heading${ level }`];
```

Comment:
I suppose so, yes. But I don't think that should be possible because you always receive the level when it's a heading, if you don't then the parser is not supposed to identify it as a heading but something else. 🤔